app/controllers/reference/ReferenceDataController.scala

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2017 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package controllers.reference
+
+import model.exchange.AssessorSkill
+import play.api.libs.json.Json
+import play.api.mvc.{ Action, AnyContent }
+import uk.gov.hmrc.play.microservice.controller.BaseController
+
+object ReferenceDataController extends ReferenceDataController
+
+trait ReferenceDataController extends BaseController {
```

Comment:
The idea is what Volodymir explained in a meeting, not sure if you had arrived by then: creating endpoints to expose from the backend reference data like skillsType, eventType, locations, etc, instead of relaying on enums in the frontend. 

Most of these are related to several controllers at the same time. I think it is better to keep it in a separate controller. I do not want the Event Controller to call AssessorController to expose the skills because actually skills can be retrieved from either Assessor pages or Event pages, although you may argue that may be we want to decouple them and have two endpoints one for event skills and another one for assessor skills.